_core/smart/tutorial-counter.md

```diff
@@ -43,7 +44,7 @@ Finally, the project dependencies are installed and your project is ready for de
 
```

Comment:
another "Hello World" reference above

Reply:
also the main structure is not reviewed anymore in the get started guide. Maybe a sentence to explain the structure makes sense. (e.g. the first tutorial does not reference tests)

Reply:
I added this to the "Getting started" topic but I was worried it would be padding the read time too much in this topic.